src/components/Main.js

```diff
@@ -1,62 +1,62 @@
 import React from 'react';
 
-
 export default function Main() {
   return (
-    <div>
-    <div style={{backgroundColor:'#08181C',minHeight: '100vh',display: 'flex',flexDirection: 'column',padding: '20px', paddingTop: '20px',}}>
-          
-      <h2 className={"text-primary"}>Overview</h2>
-      <p className={"text-secondary"}>
-        Welcome to the Deskify Help Desk website design document. This document
-        serves as a comprehensive blueprint for the construction and
-        implementation of our online support platform. Deskify is conceived as a
-        user-centric help desk solution that aims to streamline customer
-        service operations and enhance user engagement through efficient
-        ticketing and support mechanisms. This introduction outlines our vision,
-        objectives, and the scope of functionality intended for Deskify.
-      </p>
+    <body>
+      <div style={{ backgroundColor: '#08181C', width: '700px', marginLeft: '20px', padding: '20px' }}>
```

Comment:
```suggestion
      <div style={{ backgroundColor: '#08181C', minHeight: '100vh', display: 'flex', flexDirection: 'column', padding: '20px', paddingTop: '20px',}}>
```
I would keep this the same so that the list of tickets by the user would just go under all the text